app/src/main.cpp

```diff
@@ -226,51 +254,33 @@ namespace scitt
             "Failed to get host time: {}", ccf::api_result_to_str(result)));
         }
 
-        SCITT_DEBUG("Get SCITT configuration from KV store");
+        SCITT_DEBUG("Get service configuration from KV store");
         auto cfg = ctx.tx.template ro<ConfigurationTable>(CONFIGURATION_TABLE)
                      ->get()
                      .value_or(Configuration{});
 
-        ClaimProfile claim_profile;
+        SignedStatementProfile signed_statement_profile;
         cose::ProtectedHeader phdr;
         cose::UnprotectedHeader uhdr;
         try
         {
-          SCITT_DEBUG("Verify submitted claim");
-          std::tie(claim_profile, phdr, uhdr) = verifier->verify_claim(
-            body, ctx.tx, host_time, DID_RESOLUTION_CACHE_EXPIRY, cfg);
+          SCITT_DEBUG("Verify submitted signed statement");
+          std::tie(signed_statement_profile, phdr, uhdr) =
+            verifier->verify_signed_statement(
+              body, ctx.tx, host_time, DID_RESOLUTION_CACHE_EXPIRY, cfg);
         }
         catch (const verifier::VerificationError& e)
         {
-          SCITT_DEBUG("Claim verification failed: {}", e.what());
+          SCITT_DEBUG("Signed statement verification failed: {}", e.what());
           throw BadRequestError(errors::InvalidInput, e.what());
         }
-        // Retrieve current enclave measurement of this node
-        // See ccf logic in `/quotes/self`
-        std::string measurement;
```

Comment:
Is this implemented in CCF now? how can I reproduce the code that issued the receipt without the measurement?

Reply:
You can find that information in the ledger, at the transaction where the node was added. Note that the COSE signatures are not signed by node keys, but are signed by the service key instead, which is replicated among nodes.

Reply:
I've commented above

> You can find that information in the ledger

concrete steps need to be added to the docs/reproducibility.md to avoid confusion and mystery

Reply:
As explained above, this is out of scope for this change.

Reply:
https://github.com/microsoft/scitt-ccf-ledger/issues/229#issue-2605382283